CONTRIBUTING.adoc

```diff
@@ -156,6 +156,18 @@ It usually makes sense to store them in a static final field.
 
 How to write the code itself.
 
+==== Formatting
+
+This project uses https://github.com/diffplug/spotless[Spotless] to apply and verify code formatting.
```

Comment:
The documentation is slightly misleading - the command only works for Unix-based systems. Some people develop on Windows without the WSL; for those, this script will most likely not work (I can't verify I am a Linux user). Hence, we should specially mention this or also provide a windows compatible hook (although I am not sure how this will work)

Reply:
> The documentation is slightly misleading - the command only works for Unix-based systems. Some people develop on Windows without the WSL; for those, this script will most likely not work (I can't verify I am a Linux user). Hence, we should specially mention this or also provide a windows compatible hook (although I am not sure how this will work)

I don't understand this comment in total. I understand the the shell script might only run on UNIX, but the Gradle `spotless apply` task also works on Windows.

Reply:
I think the comment only refers to the `pre-commit` file.

At the moment, we could suggest that Unix users can enable it and ask Windows users to provide a PR to add a Windows version of the script.

Reply:
Yes sorry, that was misleadingly put by me, a few lines to far up